lib/modules/datasource/nextcloud/index.ts

```diff
@@ -0,0 +1,74 @@
+import { cache } from '../../../util/cache/package/decorator';
+import { regEx } from '../../../util/regex';
+import { asTimestamp } from '../../../util/timestamp';
+import * as semanticVersioning from '../../versioning/semver';
+import { Datasource } from '../datasource';
+import type { GetReleasesConfig, ReleaseResult } from '../types';
+import { Applications } from './schema';
+
+export class NextcloudDatasource extends Datasource {
+  static readonly id = 'nextcloud';
+
+  static readonly defaultTranslationLanguage = 'en';
```

Comment:
```suggestion
  private static readonly defaultTranslationLanguage = 'en';
```

not externally used, move before public members